### ISSUE TITLE
Flutter 2.8 Upgrade results in error with visibility_detector

### ISSUE THREAD

**Issue by DKAC** [zeeshux7860/FireXcode#7]
When upgrading to Flutter 2.8, the following error occurs during build. According to the visibility_detector project, this should be fixed with 0.2.2, but overriding the dev dependencies does not do the trick. 

Running "flutter pub get" in ccmob...
Warning: You are using these overridden dependencies:
! animated_interpolation 0.3.2 from git https://github.com/TheArchitect123/animated_interpolation at d4340b
! json_ast 1.0.5 from git https://github.com/DKConvista/json_ast at 376932
! visibility_detector 0.2.2
Launching lib\main.dart on Android SDK built for x86 in debug mode...
lib\main.dart:1
/C:/Users/dklink/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/firexcode-1.0.8/lib/src/Modules/visibility_detector/visibility_detector_layer.dart:276:21: Error: Too many positional arguments: 1 allowed, but 2 found.
Try removing the extra positional arguments.
    super.addToScene(builder, layerOffset);
                    ^
FAILURE: Build failed with an exception.

**Comment by matnex-mix** [comment]
did you later solve it...??

**Comment by DKAC** [comment]
No, I didn't give it another shot. Currently, 2.5 is fine with me, but tbh when it will become necessary for me to upgrade flutter, I will have another look at what else is out there that meets the requirements...
